study-builder/studies/rarex/i18n/en.json

```diff
@@ -594,6 +595,7 @@
   "qol_with_some_trouble": "With some trouble",
   "qol_with_lot_of_trouble": "With a lot of trouble",
   "qol_not_able_to_do": "Not able to do",
+  "qol_unable_to_answer_due_to_child": "Unable to answer because of my child's condition or communication skills",
```

Comment:
This pretty specific, but in DDP-5726 I am seeing the text to simply be: Unable to answer . We sure about this one?

Reply:
See DDP-6047.

> For the Quality of Life <18: for all questions related to child (start with "My child…"), change response "Unable to answer" to "Unable to answer because of my child's condition or communication skills."